cpp/util/etcd.cc

```diff
@@ -541,6 +541,10 @@ void EtcdClient::WatchInitialGetAllDone(WatchState* state, util::Status status,
 
   vector<WatchUpdate> updates;
   for (const auto& node : nodes) {
+    LOG_IF(INFO, etcd_index < node.modified_index_)
+        << "X-Etcd-Index (" << etcd_index
+        << ") smaller than node modifiedIndex (" << node.modified_index_
+        << ") for key \"" << node.key_ << "\"";
```

Comment:
yeah, WARNING might be ok; we don't watch things which change super frequently it shouldn't be too much of a hit anyway.